cli/configssh.go

```diff
@@ -316,15 +305,25 @@ func configSSH() *cobra.Command {
 				configModified = buf.Bytes()
 			}
 
-			if showDiff {
-				if len(changes) > 0 {
-					// Write to stderr to avoid dirtying the diff output.
-					_, _ = fmt.Fprint(out, "The following changes will be made to your SSH configuration:\n\n")
-					for _, change := range changes {
-						_, _ = fmt.Fprintf(out, "  * %s\n", change)
-					}
+			if len(changes) > 0 {
+				dryRunDisclaimer := ""
+				if dryRun {
+					dryRunDisclaimer = " (dry-run, no changed will be made)"
+				}
+				_, err = cliui.Prompt(cmd, cliui.PromptOptions{
+					Text:      fmt.Sprintf("The following changes will be made to your SSH configuration:\n\n    * %s\n\n  Continue?%s", strings.Join(changes, "\n    * "), dryRunDisclaimer),
+					IsConfirm: true,
+				})
```

Comment:
I don't think we need to prompt for confirmation if we're just doing a dry run

Reply:
I agree it's not strictly necessary, but I kind of wanted the behavior to be the same between `--dry-run` and normal run. For instance, this prompt shows bullet points of changes that will be made (usually just one atm), but with the diff printed out immediately after, the user might not see it (scrolls by too fast).